examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    @interact(selected_anomaly=anomalies_dropdown, delta_plotted_days=delta_plotted_days)
</pre><p>I didn't know this, very cool! Could you also check how it renders once the notebook is built in HTML for the examples documentation page? It's OK if it's not interactive, but would be good if it at least displays well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>